meetings/2023-06-07.md

```diff
@@ -54,24 +69,71 @@ URL: https://www.w3.org/2023/09/TPAC/
-* SC: Resolves https://github.com/solid/solid-wg-charter/issues/9
+URL: https://github.com/solid/solid-wg-charter/pull/38
 
+* SC: Resolves https://github.com/solid/solid-wg-charter/issues/9
+* SC: We have some agreement in the PR but there's a bit of paraphrasing we need to be careful with. I do want to wrap up the PR, but if we can try to integrate changes that would clarify. Any objections to merging this PR? 
+* ML: PAC wanted to answer but not sure if he did. 
+* SC: It is assigned to PAC. If we have agreement we would allow him to have last review and merge. 
+* SC: We can drop the word "new". We might have an exception with WebID. If it's listed as a deliverable it not intended to conflict with out of scope. 
+* SC: No objections.
 
-### Strategy for normative references
-URL: https://github.com/solid/solid-wg-charter/issues/37
+ACTION: SC to integrate change from discussion on PR with eP. PAC to have last review and merge.
 
 
 ### Consider adopting WebID 1.0 ED as a Deliverable
-https://github.com/solid/solid-wg-charter/issues/39
+URL: https://github.com/solid/solid-wg-charter/issues/39
+
+* SC: The proposal is to adopt WebID as a deliverable. We had several approvals from the CG perspective. At least one approval from WebID CG perspective. The specification is quite integral to Solid, been around for a long time, used by several communities outside Solid. It is mature. Things need to be improved, but it was deemed to be mature enough for solid ecosystem to use widely. Lot of our specs and implementations uses it. It is something we depend on. With respect to normative references, it would be in Solid WG's interest to run it through recommendation track. 
+* SC: We need agreement that we want to take it on and no objections from WebID CG.
+* JZ: Presumably this would ease the work of the Solid WebID-Profile group because we'd have something that is already on track to be accepted in the WebID spec itself. Is that correct?
+* SC: Yes.
+* JZ: Then I'm in favor. 
+* LD: This would mean we adopt the WebID spec within the WG? This would go beyond the WebID panel? Would also incorporate non-solid specific? 
+* SC: Yes. It's non-Solid specific. Solid WebID profile is one of the specs that uses it, other specs use it (AuthN, etc).
+* LD: Will we be able to maintain the boundary between the general WebID specification and the specific profile being defined for the Solid specification in the context of the Solid WG?
+* SC: WebID identity specification is strictly about identity. WebID profile is about data model.  
+* eP: +1 to adopt
+* TBL: Crucial thing is if this is something the server needs to understand it needs to be at same level of maturity as Solid protocol. Things like what's in your WebID profile should not be there because not server's concern. 
+* eP: Currently solid OIDC depends on it because it specifies the issuer must be specified in the WebID document. WebID spec is lightweight. Only has two requirements. 
```

Comment:
```suggestion
* TBL: Crucial thing is that if this is something the server needs to understand, it needs to be at same level of maturity as Solid protocol. Things like what's in your WebID profile should not be there because not server's concern. 
* eP: Currently Solid OIDC depends on it because it specifies the issuer must be specified in the WebID document. WebID spec is lightweight. Only has two requirements. 
```